posthog/tasks/status_report.py

```diff
@@ -215,9 +193,6 @@ def fetch_sql(sql_: str, params: Tuple[Any, ...]) -> List[Any]:
 
 
 def get_instance_licenses() -> List[str]:
-    if settings.EE_AVAILABLE:
```

Comment:
Sanity check question for @timgl - is EE_AVAILABLE env var safe to drop?

Reply:
I don't see why not. My only worry is what happens if someone who is on postgres tries to upgrade? Is there a way for us to warn them? I guess not